.rubocop.yml

```diff
@@ -24,6 +24,9 @@ Metrics/LineLength:
   Exclude:
     - 'app/controllers/catalog_controller.rb'
 
+Metrics/AbcSize:
+  Exclude:
+    - 'app/helpers/content_block_helper.rb'
```

Comment:
Instead of excluding this, maybe we can figure out how to simplify those methods?